### PR TITLE
https://issues.redhat.com/browse/ACM-3649--create cluster AWS needed …

### PR DESCRIPTION
**Pull request by swopebe**
…work and 2.7/2.8 update

### CODE REVIEW

**Comment by swopebe** [comment]
@Randy424

**Review by Randy424 — APPROVED** [review]
/lgtm

File: clusters/cluster_lifecycle/create_ocp_aws.adoc [hunk 1]
```diff
@@ -19,71 +20,91 @@ See the following prerequisites before creating a cluster on AWS:
...
 
-The release image identifies the version of the {ocp-short} image that is used to create the cluster. If the version that you want to use is available, you can select the image from the list of images. If the image that you want to use is not a standard image, you can enter the URL to the image that you want to use. See xref:../cluster_lifecycle/release_images.adoc#release-images[Release images] for more information about release images.
+* If there is already a base DNS domain that is associated with the selected credential that you configured with your AWS account, that value is populated in the field. You can change the value by overwriting it. This name is used in the hostname of the cluster.
 
-The node pools include the control plane pool and the worker pools. The control plane nodes share the management of the cluster activity. The information includes the following fields:
+* The release image identifies the version of the {ocp-short} image that is used to create the cluster. Select the image from the list of images that are available. If the image that you want to use is not a available, you can enter the URL to the image that you want to use.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/create_ocp_aws.adoc]:
```suggestion
* The release image identifies the version of the {ocp-short} image that is used to create the cluster. Select the image from the list of images that are available. If the image that you want to use is not available, you can enter the URL to the image that you want to use. 
```

File: clusters/cluster_lifecycle/create_ocp_aws.adoc [hunk 2]
```diff
@@ -19,71 +20,91 @@ See the following prerequisites before creating a cluster on AWS:
...
++
+** Root storage: Specify the amount of root storage allocated for your worker pool. This setting is required when you define a worker pool.
 
-* Instance type: Specify the instance type for your control plane node. You can change the type and size of your instance after it is created. 
+* Networking details are required for your cluster, and multiple networks are required for using IPv6. You can add an additional network by clicking *Add network*. 
 
-* Root storage: Specify the amount of root storage to allocate for the cluster. 
+* Proxy information that is provided in the credential is automatically added to the proxy fields. You can use the information as it is, overwrite it, or add the information if you want to enable a proxy. The following list contains the required information for creating a proxy:  
++
+* HTTP proxy: Specify the URL that should be used as a proxy for `HTTP` traffic.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/create_ocp_aws.adoc]:
```suggestion
** HTTP proxy: Specify the URL that should be used as a proxy for `HTTP` traffic. 
```

File: clusters/cluster_lifecycle/create_ocp_aws.adoc [hunk 3]
```diff
@@ -19,71 +20,91 @@ See the following prerequisites before creating a cluster on AWS:
...
 
-* Instance type: Specify the instance type for your control plane node. You can change the type and size of your instance after it is created. 
+* Networking details are required for your cluster, and multiple networks are required for using IPv6. You can add an additional network by clicking *Add network*. 
 
-* Root storage: Specify the amount of root storage to allocate for the cluster. 
+* Proxy information that is provided in the credential is automatically added to the proxy fields. You can use the information as it is, overwrite it, or add the information if you want to enable a proxy. The following list contains the required information for creating a proxy:  
++
+* HTTP proxy: Specify the URL that should be used as a proxy for `HTTP` traffic. 
++
+* HTTPS proxy: Specify the secure proxy URL that should be used for `HTTPS` traffic. If no value is provided, the same value as the `HTTP Proxy URL` is used for both `HTTP` and `HTTPS`.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/create_ocp_aws.adoc]:
```suggestion
** HTTPS proxy: Specify the secure proxy URL that should be used for `HTTPS` traffic. If no value is provided, the same value as the `HTTP Proxy URL` is used for both `HTTP` and `HTTPS`.
```

File: clusters/cluster_lifecycle/create_ocp_aws.adoc [hunk 4]
```diff
@@ -19,71 +20,91 @@ See the following prerequisites before creating a cluster on AWS:
...
++
+* No proxy sites: A comma-separated list of sites that should bypass the proxy. Begin a domain name with a period `.` to include all of the subdomains that are in that domain. Add an asterisk `*` to bypass the proxy for all destinations.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/create_ocp_aws.adoc]:
```suggestion
** No proxy sites: A comma-separated list of sites that should bypass the proxy. Begin a domain name with a period `.` to include all of the subdomains that are in that domain. Add an asterisk `*` to bypass the proxy for all destinations. 
```

File: clusters/cluster_lifecycle/create_ocp_aws.adoc [hunk 5]
```diff
@@ -19,71 +20,91 @@ See the following prerequisites before creating a cluster on AWS:
...
-* Root storage: Specify the amount of root storage to allocate for the cluster. 
+* Proxy information that is provided in the credential is automatically added to the proxy fields. You can use the information as it is, overwrite it, or add the information if you want to enable a proxy. The following list contains the required information for creating a proxy:  
++
+* HTTP proxy: Specify the URL that should be used as a proxy for `HTTP` traffic. 
++
+* HTTPS proxy: Specify the secure proxy URL that should be used for `HTTPS` traffic. If no value is provided, the same value as the `HTTP Proxy URL` is used for both `HTTP` and `HTTPS`.
++
+* No proxy sites: A comma-separated list of sites that should bypass the proxy. Begin a domain name with a period `.` to include all of the subdomains that are in that domain. Add an asterisk `*` to bypass the proxy for all destinations. 
++
+* Additional trust bundle: One or more additional CA certificates that are required for proxying HTTPS connections.
```
Comment (cadawson) [inline clusters/cluster_lifecycle/create_ocp_aws.adoc]:
```suggestion
** Additional trust bundle: One or more additional CA certificates that are required for proxying HTTPS connections.
```

**Review by cadawson — APPROVED** [review]
A few minor changes. Thanks!

**Pull request merged**